lib/database/database.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'package:firebase_database/firebase_database.dart';
+
+//https://www.youtube.com/watch?v=sXBJZD0fBa4&ab_channel=Firebase
+
+class WriteDatabase extends StatefulWidget {
+  const WriteDatabase({Key? key, required this.title}) : super(key: key);
+  final String title;
+  static const routeName = 'writeData';
+
+  @override
+  _DatabaseService createState() => _DatabaseService();
+}
+
+class _DatabaseService extends State<WriteDatabase> {
+  final database = FirebaseDatabase.instance.ref();
+
+  // This will work if copied into main.dart, unsure how to get it to work form another screen
+  @override
+  void initState() {
+    super.initState();
+    WidgetsBinding.instance?.addPostFrameCallback((timeStamp) async {
+      await database.child('/beasties').set({
```

Comment:
Is this just adding a new beastie to the db? Why would we need to add any more beasties?